### ISSUE TITLE
How to Create a SavedModel with the Conformer Model

### ISSUE THREAD

**Issue by hyunwin** [TensorSpeech/TensorFlowASR#260]
Hello!

I am pretty new to TensorFlow and have been tasked to deploy TensorFlowASR using Docker. In order to do that, I need the .pb file from the pretrained Conformer Model but don't know exactly how to obtain that. I have tried numerous excerpts of Python code that saves Keras models and creates a .pb file but none were successful. Please help me out!

**Comment by usimarit** [comment]
@hyunwin Have you tried this code: [gen_saved_model.py](https://github.com/TensorSpeech/TensorFlowASR/blob/main/examples/conformer/inference/gen_saved_model.py)?

**Comment by hyunwin** [comment]
Hello,

Yes I have. I was able to get a .pb file but I don't know if the model is pretrained or not from the gen_saved_model.py

**Comment by usimarit** [comment]
@hyunwin The script loads the trained weight [here](https://github.com/TensorSpeech/TensorFlowASR/blob/main/examples/conformer/inference/gen_saved_model.py#L52) 😄

**Comment by hyunwin** [comment]
Oh I see, so if I get the latest.h5 file from the pretrained-subword-conformer.zip, would that output the pretrained .pb file?

**Comment by usimarit** [comment]
> Oh I see, so if I get the latest.h5 file from the pretrained-subword-conformer.zip, would that output the pretrained .pb file?

Yeah basically like that. But I'm working on exporting pb for only the model.

**Comment by hyunwin** [comment]
Hi again, I was trying to serve the saved_model.pb file that I got from the Conformer model using this tutorial:
https://www.tensorflow.org/tfx/serving/docker

I haven't gotten any luck as I think I need the variables and assets folder as well in order to serve it.
